jerry-core/jerry.c

```diff
@@ -519,6 +519,103 @@ jerry_value_is_undefined (const jerry_value_t value) /**< api value */
 } /* jerry_value_is_undefined */
 
 /**
+ * Check if the specified feature is enabled.
+ *
+ * @return true  - if the specified feature is enabled,
+ *         false - otherwise.
+ */
+bool jerry_is_feature_enabled (const jerry_feature_t feature)
+{
+  switch (feature)
+  {
+    case JERRY_FEATURE_ERROR_MESSAGES:
```

Comment:
What if we use the following format?
```
switch (feature)
{
#ifdef FEATURE
    case FEATURE: return true;
#endif /* FEATURE */
...
    default: return false;
}
```
It would be less lines and probably more readable without multiple returns in one block. Also maybe we could add the case-es with a macro, just to avoid the typing the feature names multiple times. What do you think?